optuna/distributions.py

```diff
@@ -311,19 +315,20 @@ class IntLogUniformDistribution(BaseDistribution):
     This object is instantiated by :func:`~optuna.trial.Trial.suggest_int`, and passed to
     :mod:`~optuna.samplers` in general.
 
-    .. note::
-        If the range :math:`[\\mathsf{low}, \\mathsf{high}]` is not divisible by
-        :math:`\\mathsf{step}`, :math:`\\mathsf{high}` will be replaced with the maximum of
-        :math:`k \\times \\mathsf{step} + \\mathsf{low} \\lt \\mathsf{high}`, where :math:`k` is
-        an integer.
-
     Attributes:
         low:
             Lower endpoint of the range of the distribution. ``low`` is included in the range.
         high:
             Upper endpoint of the range of the distribution. ``high`` is included in the range.
         step:
             A step for spacing between values.
+
+            .. note::
+                This value is valid for only 1. Otherwise, the value is replaced with 1.
+
+            .. deprecated:: 2.0.0
+                ``step`` argument will be removed in the future. The removal of this feature is
+                currently scheduled for v4.0.0, but this schedule is subject to change.
```

Comment:
Recently, #1407 introduced `.. warning::` instead of `.. deprecated::` to show the deprecation as follows:

![image](https://user-images.githubusercontent.com/3255979/85557200-86a17580-b662-11ea-939c-6b5ffbba4793.png)
([link](https://optuna.readthedocs.io/en/latest/reference/structs.html#optuna.structs.StudySummary))

Could you update the style of this message using `.. warning::`?

Reply:
Thank you for your comment! Sure. The current part of `step` shows 
![Screenshot 2020-06-25 at 0 24 36](https://user-images.githubusercontent.com/7121753/85584941-8e6d1400-b67a-11ea-91e2-067e01ac70f1.png)